nservicebus/nhibernate/accessing-data_customizing-session_nhibernate_[6,7).partial.md

```diff
@@ -0,0 +1,7 @@
+## Customizing the session
+
+Versions 5 and below supported customizing the instantiation of the `ISession`. This is done by hooking up to the creation process by providing a custom delegate:
```

Comment:
@SzymonPobiega I fixed the partial to be more precise about the fact that only version 6 allows to customize creation of `ISession`.